internal/pkg/upgrade/upgrade.go

```diff
@@ -20,6 +21,7 @@ const (
 	dtmBakFileName = "dtm-bak"
 	dtmOrg         = "devstream-io"
 	dtmRepo        = "devstream"
+	dtmExecute     = "dtm"
```

Comment:
The filename with DevStream cli file may not be "dtm". Users could rename it to any.

Here's some hint for you:

```go
binFilePath, err := os.Executable()
dtmFileDir, dtmFileName = filepath.Split(binFilePath)
```

Reply:
This has been resolved in #1133 .

@Shuimo03 Please use this variable:

https://github.com/devstream-io/devstream/blob/30cf3afdf2bea912217ff33a3b770a0dea6d1b72/internal/pkg/upgrade/upgrade.go#L25-L26

Reply:
thank you, I try variable